### PR TITLE
Implement namespace support for exceptions

### PR DESCRIPTION
**Pull request by mblaschke**
Add namespace support for exceptions for shared clusters.

### CODE REVIEW

File: pkg/config/exemptions.go [hunk 1]
```diff
@@ -14,10 +15,16 @@ func (conf Configuration) IsActionable(ruleID, controllerName string) bool {
 	}
 
 	for _, example := range conf.Exemptions {
+		if example.Namespace != "" && example.Namespace != namespace {
+			fmt.Println("not matches: " + example.Namespace + " -> " + namespace)
```
Comment (rbren) [inline pkg/config/exemptions.go:19]: I think we can get rid of this `Println`

**Review by rbren — APPROVED** [review]
Great work, and thanks for adding a test! Just one little note

**Comment by rbren** [comment]
Can you also add documentation here: https://github.com/FairwindsOps/polaris/blob/master/docs/usage.md#exemptions

File: pkg/config/exemptions.go (hunk 1 above)
Comment (mblaschke) [reply pkg/config/exemptions.go]: ah, sure.. fixed :)

**Comment by mblaschke** [comment]
Done

**Comment by marcemv90** [comment]
With this new feature is it possible to ignore completely a namespace? Because I would like to get rid of the kube-system namespace in the dashboard.

**Review by baderbuddy — APPROVED** [review]
This looks really good to me! Thank you!

**Comment by baderbuddy** [comment]
@marcemv90 this should allow all controllers within a namespace to be exempted (since it matches on prefix you could add a controller-name of `""`) but you'd have to specify the rules to exempt still

**Pull request merged**